website/src/pages/docs/features/automatic-persisted-queries.mdx

```diff
@@ -119,6 +119,36 @@ server.listen(4000, () => {
 
 For external stores the `set` and `get` properties on the store can also return a `Promise`.
 
+In production, it's recommended to capture the errors from any store that could stop functioning.
+Instead of raising an error, returning undefined or null will allow the server to continue to
+respond to requests if the store goes down.
+
+```ts filename="Automatic Persisted Operations with a redis store" {16}
+import Keyv from "keyv";
+
+const store = new Keyv("redis://user:pass@localhost:6379");
+
+useAPQ({
+  store: {
+    async get(key) {
+      try{
+        return await store.get(key);
+      } catch(e) {
+        console.error(`Error while fetching the operation: ${key}`, e);
+      }
+    },
+    async set(key, value) {
+      try{
+        return await store.set(key, value);
+      } catch(e) {
+        console.error(`Error while saving the operation: ${key}`, e);
+      }
+    }
+  }
+})
+```
```

Comment:
I think this could be worth to put in a callout to make it more visible.

Not sure if the code have to be in the Callout, but since the implications of this are pretty important, I think it have to be very visible.

```suggestion
<Callout>
	In production, it's recommended to capture the errors from any store that could stop functioning.
	Instead of raising an error, returning undefined or null will allow the server to continue to
	respond to requests if the store goes down.
	
	```ts filename="Automatic Persisted Operations with a redis store" {16}
	import Keyv from "keyv";
	
	const store = new Keyv("redis://user:pass@localhost:6379");
	
	useAPQ({
	  store: {
	    async get(key) {
	      try{
	        return await store.get(key);
	      } catch(e) {
	        console.error(`Error while fetching the operation: ${key}`, e);
	      }
	    },
	    async set(key, value) {
	      try{
	        return await store.set(key, value);
	      } catch(e) {
	        console.error(`Error while saving the operation: ${key}`, e);
	      }
	    }
	  }
	})
	```
</Callout>
```

Reply:
Added callout